build.gradle.kts

```diff
@@ -14,26 +14,26 @@ import java.util.zip.ZipEntry
 import java.util.zip.ZipOutputStream
 
 plugins {
-    kotlin("jvm") version "1.7.0"
-    id("org.jetbrains.intellij") version "1.7.0"
-    id("org.sonarqube") version "3.3"
+    kotlin("jvm") version "1.7.10"
+    id("org.jetbrains.intellij") version "1.9.0"
+    id("org.sonarqube") version "3.4.0.2513"
     java
     jacoco
     id("com.github.hierynomus.license") version "0.16.1"
-    id("com.jfrog.artifactory") version "4.24.20"
-    id("com.google.protobuf") version "0.8.17"
+    id("com.jfrog.artifactory") version "4.29.0"
+    id("com.google.protobuf") version "0.8.19"
     idea
     signing
-    id("de.undercouch.download") version "4.1.2"
-    id("org.cyclonedx.bom") version "1.5.0"
+    id("de.undercouch.download") version "5.2.0"
+    id("org.cyclonedx.bom") version "1.7.1"
 }
 
 buildscript {
     repositories {
         mavenCentral()
     }
     dependencies {
-        "classpath"(group = "org.jetbrains.intellij", name = "blockmap", version = "1.0.5")
+        "classpath"(group = "org.jetbrains.intellij", name = "blockmap", version = "1.0.6")
```

Comment:
I noticed the other day that we are building these blockmap files. I don't think we push them to the marketplace, do we still need to keep that in the build?

Reply:
They are normally automatically computed by the marketplace. Here the goal was an attempt to have incremental update for our dogfood channel from repox. I have never been convinced it is actually working. We would have to spend some time on it.